package.json

```diff
@@ -117,7 +117,7 @@
     "@elastic/ecs": "^8.11.1",
     "@elastic/elasticsearch": "^8.15.0",
     "@elastic/ems-client": "8.5.3",
-    "@elastic/eui": "97.3.1",
```

Comment:
Can we also update https://github.com/elastic/kibana/blob/8.17/src/dev/license_checker/config.ts#L90